src/actions/localisationActions.js

```diff
@@ -60,65 +62,82 @@ type SetLngAndBundle = {
   onSuccess?: () => void,
 }
 
-const getTranslationData = (lng: string) => localeConfig.namespaces.map((ns) => ({
-  ns,
-  url: `${localeConfig.baseUrl}${lng}/${ns}.json`, // eslint-disable-line i18next/no-literal-string
-}));
-
-const getCachedTranslationResources =
-  async (translationsData: TranslationData[], cachedUrls: CachedUrls, dispatch: Dispatch) => {
-    const cachedTranslations = await Promise.all(translationsData.map(async ({ ns, url }) => {
-      const { localPath } = cachedUrls?.[url] || {};
-      if (!localPath) return { ns, translations: {} };
-
-      const translations = await getCachedJSONFile(localPath);
-
-      if (!translations) {
-        // cached file no longer exists - remove it from map
-        dispatch(removeUrlCacheAction(url));
-        return { ns, translations: {} };
-      }
-      return { ns, translations };
-    })).catch((e) => {
-      reportLog(LANGUAGE_ERROR.NO_TRANSLATIONS, e);
-      return [];
-    });
-
-    return cachedTranslations.reduce((formattedResources, translation) => {
-      const { ns, translations } = translation;
-      if (ns && translations) formattedResources[ns] = translations;
-      return formattedResources;
-    }, {});
+const getTranslationData = (lng: string, baseUrl: string, version: string) => localeConfig.namespaces.map((ns) => {
+  let url = '';
+  if (baseUrl) {
+    /* eslint-disable i18next/no-literal-string */
+    if (version) {
+      url = `${baseUrl}${lng}/${ns}_${version}.json`;
+    } else {
+      url = `${baseUrl}${lng}/${ns}.json`;
+    }
+    /* eslint-enable i18next/no-literal-string */
+  }
+  return {
+    ns,
+    url,
   };
+});
+
+type GetTranslationResourcesProps = {
+  language: string,
+  dispatch: Dispatch,
+  getState: GetState,
+  getLocal?: boolean,
+}
 
-const getTranslationsResources = async (props) => {
+const getTranslationsResources = async (props: GetTranslationResourcesProps) => {
   const {
     language,
     dispatch,
     getState,
+    getLocal = false,
   } = props;
   let resources;
-  // TODO: pass in versioning;
   let version = '';
+  let baseUrl = '';
+
+  try {
+    version = firebaseRemoteConfig.getString(FEATURE_FLAGS.APP_LOCALES_LATEST_TIMESTAMP);
+    baseUrl = firebaseRemoteConfig.getString(FEATURE_FLAGS.APP_LOCALES_URL);
+  } catch (e) {
```

Comment:
Just wondering if the try / catch is necessary here? I'm not sure that `getString` will throw an Error if the value isn't found...

Reply:
It prevented crashing if firebaseRemoteConfig wasn't yet initiated. 
I'm waiting for it to have default values set in order to execute any initial app actions. So this check is no longer needed.